pkg/network/protocols/http/testutil/testutil.go

```diff
@@ -48,6 +51,20 @@ func HTTPServer(t *testing.T, addr string, options Options) func() {
 		io.Copy(w, req.Body)
 	}
 
+	/* Save and recover TCP timestamp option */
+	oldTCPTS, err := sysctl.Get("net.ipv4.tcp_timestamps")
```

Comment:
the change here mean that we must run all tests in SUDO mode, in the past other developers have complained about that. Can you check that we're in sudo mode before running those command lines?
and generally, the change will affect all tests that uses HTTPServer, I wonder if we could make sure we're not affecting other tests.
Options:
1. change `EnableTCPTimestamp` to `*bool` and only if it is not nil, then to apply the change
2. split into 2 different functions

what do you think?

Reply:
I added helper, and skip the running test if it tried to set a different value than the os and don't have privileges

all sysprobe tests are runnng under sudo/root
The previous issue we had was package not under linux_bpf require sudo for the test itself (unshare or sudo -u)